core/src/main/java/org/fao/geonet/kernel/DataManager.java

```diff
@@ -2066,9 +2066,13 @@ public Metadata updateMetadata(final ServiceContext context, final String metada
         Integer intId = Integer.valueOf(metadataId);
         if (ufo) {
             String parentUuid = null;
-            metadataXml = updateFixedInfo(schema, Optional.of(intId), null, metadataXml, parentUuid, (updateDateStamp ? UpdateDatestamp
-
-                    .YES : UpdateDatestamp.NO), context);
+            metadataXml = updateFixedInfo(schema,
```

Comment:
Please revert the identation optimisation, to stay closer from upstream.
thanks